readme.md

```diff
@@ -28,3 +28,17 @@ NEXT_PUBLIC_PROJECT_ID="your_project_id"
 ```zsh
 VITE_PROJECT_ID="your_project_id"
 ```
+
+# Releasing new versions
+
+1. Merge your feature branch into `V3`
+2. Create a new branch from `V3` and name it with the version tag
+3. Run `npm run outdated` and update dependencies
```

Comment:
Should this be `npm outdated` instead `npm run outdated` since we don't have any script with that name.